### ISSUE TITLE
Update editor template

### ISSUE THREAD

**Issue by lwasser** [pyOpenSci/software-peer-review#200]
After a convo with @NickleDave and @cmarmo i think it would be good to update our editor because it's very easy to forget things.
So it shoudl have a list of tasks the editor should check before continuing this will also allow. me to parse data to determine when the review began.

<img width="760" alt="Screen Shot 2023-03-10 at 2 25 49 PM" src="https://user-images.githubusercontent.com/7649194/224431439-6d85947d-1d9b-4616-9a5d-dd1248e303e0.png">
 templ

We should add to this list checkboxes for:

* editor in chief checks are complete (this is just a nice cross check step for editors to do!) and sometimes an editor may be asked to perform that check. 
* Pre-review survey has been completed by all package authors, and all reviewers and editors on the issue (if the editor has already taken the survey once they do NOT need to take it again). 
* Header info at the top of the issue has been updated with reviewer and editor names
* issue label has been updated with "reviewers assigned"

Post review - we should add the following to the template

* reviewers and package author have taken the post review survey
* Editor has added all people involved to the readme via all-contributors bot 
* 
* then the other items already there!

Anything that i'm missing here @cmarmo @NickleDave ?? many thanks for asking questions about this process - i do think this will help greatly :)

**Comment by NickleDave** [comment]
I don't see anything missing. I think there were some to-dos for the reviewer section of the guide too that came up in Slack--I raised a separate issue about minor tweaks to that, just to not overload this one.

**Comment by lwasser** [comment]
ok wonderful!! ... have a look at this - i kind of rewrote things in our [pynteny review
](https://github.com/pyOpenSci/software-submission/issues/67#issuecomment-1464509583)

David this is where we can remove the step of the author and reviewers adding themselves in the future once my issue parsing workflow is working (it almost is now). 

```
# Pynteny is now a part of the pyOpenSci Ecosystem!
----------------------------------------------
🎉 Pynteny has been approved by pyOpenSci! Thank you @Robaina  for submitting pyteny and many thanks to @Batalex @c-thoben for reviewing this package! 😸 NOTE:  I have to say this was an extraordinary review over all. @NickleDave you also really did a wonderful job in getting this review together early on. All aspects of this review went exactly as i would hope a pyOpenSci review would go!! We want our reviews to be a conversation that **supports** maintainers in improving their package! And i really think this was achieved here. So thank you ALL for being a tremendous team in this review. 

There are a few things left to do to wrap up this submission after you've taken the post review survey! 

## Author Tasks 

@Robaina please do the following:
- [ ] Activate [Zenodo](https://zenodo.org/) watching the repo if you haven't already done so. (please ask if this if you have any questions about this!)
- [ ] Tag and create a release to create a Zenodo version and DOI. Please share the DOI here so either I can update the archive at the top of this issue!
- [ ] Add the badge for pyOpenSci peer-review to <PACKAGE-NAME-HERE> README.md file. The badge should be `[![pyOpenSci](https://tinyurl.com/y22nb8up)](https://github.com/pyOpenSci/software-review/issues/issue-number)`. 
- [ ] Add <package-name> to the pyOpenSci website. <author-github-handle>, please open a pr to update [this file](https://github.com/pyOpenSci/pyopensci.github.io/blob/main/_data/packages.yml): to add your package and name to the list of contributors.

## Editor Tasks 
- [ ] Please add everyone involved in this review to our contributors list using the **all contributors** bot
_NOTE: unfortunately you will need to do this one contributor at a time and merge the pr that is opened each time._ 
Example comment to do this: `<tag> all-contributors please add @robaina for code, review`
- [ ] Please double check that the author added the badge to the readme and the link from the badge is to this issue.
- [ ] Please double check that both reviewers and the author completed our [post review survey](https://forms.gle/6WG8hPzYpyUYJg2B7)! 
- [ ] If joss - please make sure that the JOSS submission is fast tracked (no second review is required)

<IF JOSS SUBMISSION>
It looks like you would like to submit this package to JOSS. Here are the next steps:

- [ ] Login to the JOSS website and fill out the JOSS submission form using your Zenodo DOI. **When you fill out the form, be sure to mention and link to the approved pyOpenSci review.** JOSS will tag your package for expedited review if it is already pyOpenSci approved.
- [ ] Wait for a JOSS editor to approve the presubmission (which includes a scope check).
- [ ] Please add a link to the JOSS review issue once it is created here as well
- [ ] Once the package is approved by JOSS, you will be given instructions by JOSS about updating the citation information in your README file.
- [ ] When the JOSS review is complete, add a comment to your review here in the pyOpenSci software-review repo that it has been approved by JOSS.

🎉 Congratulations! You are now published with both JOSS and pyOpenSci! 🎉
<IF JOSS SUBMISSION/>

All -- if you have any feedback for us about the review process please feel free to share it here. We are always looking to improve our process and documentation in the [software-peer-review-guide](https://www.pyopensci.org/software-peer-review). We have also been updating our documentation to improve the process, so all feedback is appreciated!

```

**Comment by lwasser** [comment]
is this too much? i just suspect we need to be really clear about the steps in each template. 
the other question is do we want to track the DOI from JOSS in our issues too?

**Comment by lwasser** [comment]
ok we also need to add the post-review survey to the [summary checklist that the editor posts here](https://www.pyopensci.org/software-peer-review/how-to/editors-guide.html#how-to-accept-a-package-into-the-pyopensci-ecosystem)

**Comment by lwasser** [comment]
this actually was implemented and merged. But, this issue was never closed. i'll close it now!
